gensrc/proto/cloud.proto

```diff
@@ -1095,6 +1095,8 @@ message AlterClusterRequest {
     optional string cloud_unique_id = 2; // For auth
     optional ClusterPB cluster = 3;
     optional Operation op = 4;
+    // for SQL mode rename cluster, rename to cluster name eq instance empty cluster name, need drop empty cluster
+    optional bool drop_empty_cluster = 5 [default = false];
```

Comment:
do not use default  semantic, just make it optional is enough.
suggest naming `replace_if_existing_empty_target_cluster`